app/src/main/java/com/github/palFinderTeam/palfinder/meetups/fragments/CriterionsFragment.kt

```diff
@@ -72,6 +84,7 @@ class CriterionsFragment(val viewModel: MeetUpCreationViewModel) : DialogFragmen
                 getString(R.string.radio_male_and_female) -> viewModel.setCriterionGender(CriterionGender.ALL)
             }
 
+            // set max value to criterionAge if selected is '66+'

```

Comment:
why no setting 0 or -1 instead of max value ? Or using the MAX_AGE ?

Reply:
The slider goes only to '66+', so we would like to accept also the users aged more than 66, and the MaxValue of int provides that without checking with a 'if' in the canJoin function